src/app.ts

```diff
@@ -13,14 +13,40 @@ interface TemperatureSummary {
   average: number
 }
 
+let reading: TemperatureReading[];
+
 export function processReadings(readings: TemperatureReading[]): void {
-  // add here your code
+  reading = readings;
```

Comment:
I noticed you're processing in `getTemperatureSummary'. But, what is more expensive `processing once and reading multiple times` or `processing every time you read`?